ch32fun/ch32fun.ld

```diff
@@ -65,9 +65,20 @@ MEMORY
 	FLASH (rx) : ORIGIN = 0x00000000, LENGTH = 32K
 	RAM (xrw) : ORIGIN = 0x20000000, LENGTH = 6K
 #elif TARGET_MCU_LD == 7
-	/* CH32V005, CH32V006, CH32V007 */
-	FLASH (rx) : ORIGIN = 0x00000000, LENGTH = 62K
-	RAM (xrw) : ORIGIN = 0x20000000, LENGTH = 8K
+  /* CH32V005, CH32V006, CH32V007 */
```

Comment:
Tabs in this file not spaces.

Reply:
[ea55eee](https://github.com/cnlohr/ch32fun/pull/534/commits/ea55eeebf7c5cb382c77407d0791805015001375)